pkg/datapath/linux/config/config.go

```diff
@@ -476,6 +476,10 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 		cDefinesMap["ENABLE_IDENTITY_MARK"] = "1"
 	}
 
+	if option.Config.EnableCustomCalls {
+		cDefinesMap["ENABLE_CUSTOM_CALLS"] = "1"
```

Comment:
Could you also add this macro to `bpf/Makefile`, for the complexity tests and to be compile tested?